src/util/maps.js

```diff
@@ -5,41 +5,67 @@ import config from '../config';
 
 const { LatLng } = sdkTypes;
 
+const EARTH_RADIUS = 6371000; /* meters  */
+const DEG_TO_RAD = Math.PI / 180.0;
+const THREE_PI = Math.PI * 3;
+const TWO_PI = Math.PI * 2;
+
+const degToRadians = latlng => {
+  const { lat, lng } = latlng;
+  const latR = lat * DEG_TO_RAD;
+  const lngR = lng * DEG_TO_RAD;
+  return { lat: latR, lng: lngR };
+};
+
+const radToDegrees = latlngInRadians => {
+  const { lat: latR, lng: lngR } = latlngInRadians;
+  const lat = latR / DEG_TO_RAD;
+  const lng = lngR / DEG_TO_RAD;
+  return { lat, lng };
+};
+
 /**
  * This obfuscatedCoordinatesImpl function is a temporary solution for the coordinate obfuscation.
  * In the future, improved version needs to have protectedData working and
  * available in accepted transaction.
+ *
+ * Based on:
+ * https://gis.stackexchange.com/questions/25877/generating-random-locations-nearby#answer-213898
```

Comment:
I truly wish we had better sources for reference than a random answer with three upvotes :/ But I hope you tested this well, no need to change if it works.